src/components/themed/WalletListSwipeRow.js

```diff
@@ -70,7 +76,7 @@ function WalletListSwipeRowComponent(props: Props) {
       guiWallet != null &&
       // Some wallets launch an activation screen when selected,
       // so avoid going to the transaction list in that case:
-      (isToken || !getSpecialCurrencyInfo(guiWallet.type).isAccountActivationRequired || guiWallet.receiveAddress.publicAddress)
+      (isToken || !getSpecialCurrencyInfo(edgeWallet.type).isAccountActivationRequired || walletAddress !== '')
```

Comment:
This is probably not the best solution. The real issue is that sometimes `dispatch(selectWallet)` on the line above will launch the wallet activation scene. In that case, we shouldn't launch the transaction list scene too. If the `selectWallet` action could be modified to return a boolean, we could avoid loading the wallet address altogether in this component.

Reply:
This is a good point, but seems out of scope for this change. I don't believe we have a `walletActivated` boolean in the plugin or core. We are for now relying on the wallet having an address to determine activation.